scripts/jest.js

```diff
@@ -1,8 +1,9 @@
 const jest = require('jest');
-const config = require('./ui_framework_test_config');
+const config = require('../test/jest/config');
```

Comment:
The separation between grunt and script makes sense to me as I see them (scripts and grunt tasks) as separate parts of the overall project, but I don't see the benefit of separating out the config from the script itself unless the configuration could be swapped out at runtime, which is not the case here.  We do this separation in the grunt task folder structure as well, and it's made it pretty confusing to navigate and make sense of what is happening at times.